sklearn/metrics/cluster/_supervised.py

```diff
@@ -953,7 +953,6 @@ def adjusted_mutual_info_score(
         return 1.0
 
     contingency = contingency_matrix(labels_true, labels_pred, sparse=True)
-    contingency = contingency.astype(np.float64, copy=False)
```

Comment:
This conversion was indeed useless. At this point contengency has np.int64 dtype, which is the optimal dtype for emi now